src/components/calcite-shell-center-row/calcite-shell-center-row.scss

```diff
@@ -60,3 +64,21 @@
   width: 100%;
   height: 100%;
 }
+
+::slotted(calcite-action-bar[position="start"]) {
+  border-right: 1px solid var(--calcite-app-border);
+}
+::slotted(calcite-action-bar[position="end"]) {
+  border-left: 1px solid var(--calcite-app-border);
+}
+
+.calcite--rtl {
+  ::slotted(calcite-action-bar[position="start"]) {
+    border-right:none;
+    border-left: 1px solid var(--calcite-app-border);
+  }
+  ::slotted(calcite-action-bar[position="end"]) {
+    border-left: none;
+    border-right: 1px solid var(--calcite-app-border);
+  }
+}
```

Comment:
Could you run Prettier on this file? It should add a trailing newline and ensure space between props and values.